Makefile

```diff
@@ -339,6 +339,29 @@ delete-cluster: $(CLUSTERCTL) ## Deletes the example Kubernetes Cluster "cluster
 kind-reset: ## Destroys the "clusterapi" kind cluster.
 	kind delete cluster --name=clusterapi || true
 
+.PHONY: create-debug-cluster
+create-debug-cluster: $(CLUSTERCTL) $(ENVSUBST) ## Create a development Kubernetes cluster on Azure in a KIND management cluster.
```

Comment:
why do you need this? Can [tilt](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/development.md#tilt-for-dev-in-capz) or the[ /hack/create-dev-cluster.sh script ](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/development.md#creating-a-dev-cluster) methods meet your debugging needs?

Reply:
I would like a kind cluster with everything installed except the manager, so i can then run the manager locally pointing to the cluster. I'll remove the make target and  add an option to allow not deploying the manager.

Reply:
I removed this target and made the existing one configurable